infra/charts/management-service/README.md

```diff
@@ -0,0 +1,99 @@
+| xpManagement.apiConfig | object | `{}` | XP API server configuration.  |
+| xpManagement.extraArgs | list | `[]` | List of string containing additional XP API server arguments. For example, multiple "-config" can be specified to use multiple config files |
+| xpManagement.extraEnvs | list | `[]` | List of extra environment variables to add to XP API server container |
+| xpManagement.extraLabels | object | `{}` | List of extra labels to add to XP API K8s resources |
+| xpManagement.extraVolumeMounts | list | `[]` | Extra volume mounts to attach to XP API server container. For example to mount the extra volume containing secrets |
+| xpManagement.extraVolumes | list | `[]` | Extra volumes to attach to the Pod. For example, you can mount  additional secrets to these volumes |
+| xpManagement.image.pullPolicy | string | `"IfNotPresent"` | Docker image pull policy |
+| xpManagement.image.registry | string | `"docker.io/"` | Docker registry for XP API image |
+| xpManagement.image.repository | string | `"xp-management"` | Docker image repository for XP API |
+| xpManagement.image.tag | string | `"latest"` | Docker image tag for XP API |
+| xpManagement.ingress.class | string | `""` | Ingress class annotation to add to this Ingress rule,  useful when there are multiple ingress controllers installed |
+| xpManagement.ingress.enabled | bool | `false` | Enable ingress to provision Ingress resource for external access to XP API |
+| xpManagement.ingress.host | string | `""` | Set host value to enable name based virtual hosting. This allows routing HTTP traffic to multiple host names at the same IP address. If no host is specified, the ingress rule applies to all inbound HTTP traffic through  the IP address specified. https://kubernetes.io/docs/concepts/services-networking/ingress/#name-based-virtual-hosting |
+| xpManagement.labels | object | `{}` |  |
+| xpManagement.livenessProbe.initialDelaySeconds | int | `60` | Liveness probe delay and thresholds |
+| xpManagement.livenessProbe.path | string | `"/v1/internal/live"` | HTTP path for liveness check |
+| xpManagement.livenessProbe.periodSeconds | int | `10` |  |
+| xpManagement.livenessProbe.successThreshold | int | `1` |  |
+| xpManagement.livenessProbe.timeoutSeconds | int | `5` |  |
+| xpManagement.readinessProbe.initialDelaySeconds | int | `60` | Liveness probe delay and thresholds |
+| xpManagement.readinessProbe.path | string | `"/v1/internal/ready"` | HTTP path for readiness check |
+| xpManagement.readinessProbe.periodSeconds | int | `10` |  |
+| xpManagement.readinessProbe.successThreshold | int | `1` |  |
+| xpManagement.readinessProbe.timeoutSeconds | int | `5` |  |
+| xpManagement.replicaCount | int | `1` |  |
+| xpManagement.resources | object | `{}` | Resources requests and limits for XP API. This should be set  according to your cluster capacity and service level objectives. Reference: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ |
```

Comment:
Now that we've renamed the values, should we call it "XP Management Service" instead of "XP API" here and in other places where appropriate?